src/main/resources/application.yaml

```diff
@@ -2,18 +2,21 @@ spring:
   application:
     name: mod-tags
   datasource:
-    continue-on-error: true
+    spring:
+      sql:
+        init:
+          continue-on-error: true
```

Comment:
I think we agreed to rely on spring migrator in order not to change configs. Anyway, this does not look correct(double "spring" )